ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitter.scala

```diff
@@ -71,23 +71,16 @@ private[kvutils] class TransactionCommitter(
     "validate_contract_keys" -> validateContractKeys,
     "validate_model_conformance" -> validateModelConformance,
     "blind" -> blind,
-    "remove_unnecessary_nodes" -> removeUnnecessaryNodes,
+    "trim_unnecessary_nodes" -> trimUnnecessaryNodes,
```

Comment:
This rename just removes an annoying IntelliJ Idea warning about mutators without parentheses.

Reply:
You can probably configure the IntelliJ inspections to make sure it's not raised.

Reply:
True but I'd rather people didn't bump into this annoyance, rather than adding an IDE specific comment or having everyone configure it.